bigbluebutton/greenlight
# Use the user image for BBB avatar #2 (Limit image size)

## Description
This is basically a modified version of #2308, passing the user image of GL to BBB for displaying it as the avatar. However this version limits the size of the avatar so that BBB can avoid being crashed by somebody sending a huge image.

The maximum size can be set by the parameter AVATAR_SIZE_LIMIT in .env file. 

This PR is sent as a workaround until GL implements a more sophisticated system for uploading images.

## Testing Steps
1. Start a meeting with the user image set on GL.
2. See the avatar on BBB.
3. Another user joins the meeting with the user image set on GL
4. See his avatar as well.


## Screenshots (if appropriate):
